src/iojunk.jl

```diff
@@ -51,7 +51,7 @@ end
 
 # Read and return a byte from f. Throws EOFError if there is no more byte to read.
 function read(f::ReadableFile, ::Type{UInt8})
-	# This function needs to be fast because readbytes, readall, etc.
+	# This function needs to be fast because readbytes, readstring, etc.
```

Comment:
FWIW, this package is still claiming 0.3 support, which is why I don't want to change it too much..... Maybe it's fine to drop 0.3 support? @fhs